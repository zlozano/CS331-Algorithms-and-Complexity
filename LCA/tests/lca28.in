11 () (7)
10 (12) (6)
12 () (10)
1 (2) ()
3 (5,6) (2)
2 (3,4) (1)
5 (9) (3)
4 (7,8) (2)
7 (11) (4)
6 (10) (3)
9 () (5)
8 () (4)
