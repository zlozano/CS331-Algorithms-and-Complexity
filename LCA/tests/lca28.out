 1  1  1  1  1  1  1  1  1  1  1  1
 1  2  2  2  2  2  2  2  2  2  2  2
 1  2  3  2  3  3  2  2  3  3  2  3
 1  2  2  4  2  2  4  4  2  2  4  2
 1  2  3  2  5  3  2  2  5  3  2  3
 1  2  3  2  3  6  2  2  3  6  2  6
 1  2  2  4  2  2  7  4  2  2  7  2
 1  2  2  4  2  2  4  8  2  2  4  2
 1  2  3  2  5  3  2  2  9  3  2  3
 1  2  3  2  3  6  2  2  3 10  2 10
 1  2  2  4  2  2  7  4  2  2 11  2
 1  2  3  2  3  6  2  2  3 10  2 12
